internal/storage/storage_handle_test.go

```diff
@@ -23,202 +23,200 @@ import (
 
 	mountpkg "github.com/googlecloudplatform/gcsfuse/v2/internal/mount"
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/storage/storageutil"
-	"github.com/jacobsa/oglematchers"
 	. "github.com/jacobsa/ogletest"
+	"github.com/stretchr/testify/suite"
 )
 
 const invalidBucketName string = "will-not-be-present-in-fake-server"
 const projectID string = "valid-project-id"
 
-func TestStorageHandle(t *testing.T) { RunTests(t) }
-
 type StorageHandleTest struct {
+	suite.Suite
 	fakeStorage FakeStorage
 }
 
-var _ SetUpInterface = &StorageHandleTest{}
-var _ TearDownInterface = &StorageHandleTest{}
-
-func init() { RegisterTestSuite(&StorageHandleTest{}) }
+func TestStorageHandleSuite(t *testing.T) {
+	suite.Run(t, new(StorageHandleTest))
+}
 
-func (t *StorageHandleTest) SetUp(_ *TestInfo) {
+func (suite *StorageHandleTest) SetupSuite() {
 	var err error
-	t.fakeStorage = NewFakeStorage()
-	AssertEq(nil, err)
+	suite.fakeStorage = NewFakeStorage()
```

Comment:
any reason we are using suite. instead of assert. one of the usp of this library is rich assert package, see [ref](https://github.com/stretchr/testify?tab=readme-ov-file#assert-package) here, any reason why we are not leveraging it ? asserts are commonly used in unit tests in all languages if both suite and assert are having same functionality, should prefer asserts.